coops-ui/src/components/Dashboard/CoaWarningDialog.vue

```diff
@@ -11,7 +11,7 @@
       <v-card-actions>
         <v-btn color="primary" text @click="toggle()">Return to dashboard</v-btn>
       <v-spacer></v-spacer>
-        <v-btn color="primary" text @click="proceed()">Continue to Change of Address</v-btn>
+        <v-btn id="btn-proceed-coa" color="primary" text @click="proceed()">Continue to Change of Address</v-btn>
```

Comment:
Is this an id for QA? If so, cool. And could you also add a id to the toggle button?

Reply:
It is indeed, ok will do!